drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -1212,8 +1213,15 @@ def _get_item_stats(self):
         """
         items = self._get_item_raw_stats()
 
-        correct_count = len(items.correctly_placed) + len(items.decoy_in_bank)
-        total_count = len(items.required) + len(items.decoy)
+        correct_count = len(items.correctly_placed)
+        total_count = len(items.required)
+
+        if hasattr(self.scope_ids.usage_id, 'context_key') and \
```

Comment:
@pkulkark, we may not need to check if this attribute exists, because it should be created by default.

Reply:
It turns out that it may not be trivial to handle ID generation automatically for the versions of `XBlock` and `xblock-sdk` used here. Therefore, let's do the following as a temporary workaround:
1. Change this back to `runtime.course_id`. Keep the `hasattr` check because integration tests won't have this.
2. Add `runtime.course_id = "dummy_course_id"` to the [`make_block`](https://github.com/openedx/xblock-drag-and-drop-v2/blob/master/tests/utils.py#L27-L27) function.
3. Enable the Waffle flag for new unit tests with `@patch('drag_and_drop_v2.drag_and_drop_v2.get_grading_ignore_decoys_waffle_flag', lambda: Mock(is_enabled=lambda _: True))`.

Reply:
Thanks @Agrendalath. Updated.